### ISSUE TITLE
question regarding `bulk_long_pipeline`

### ISSUE THREAD

**Issue by sparthib** [OliverVoogd/FLAMES#49]
Hi there,

Just wondering if you have thoughts on using tools to orient bulk reads before alignment with minimap2. From what I understand in the FLAMES documentation, reads from the FASTQ are directly alignment with minimap2, whereas in the single-cell pipeline BLAZE is used to demux and orient the strands. Is there a reason strand orientation hasn't been implemented for bulk? How would this work for direct cDNA, and if you have thoughts on minimap2 settings for unstranded ONT direct cDNA reads that would be great. 

Thanks,

Sowmya

**Comment by ChangqingW** [comment]
Good point. Also we have not applied the strand option in minimap2. We are planning on adding this.
